pandapipes/component_models/abstract_models/branch_models.py

```diff
@@ -130,26 +130,23 @@ def calculate_derivatives_thermal(cls, net, branch_pit, node_pit, idx_lookups, o
 
         transient = get_net_option(net, "transient")
 
-        tvor = branch_pit[:, T_OUT_OLD]
+        tvor = branch_component_pit[:, T_OUT_OLD]
 
         delta_t = get_net_option(net, "dt")
 
         if transient:
             t_m = t_init_i1  # (t_init_i1 + t_init_i) / 2
+
             branch_component_pit[:, LOAD_VEC_BRANCHES_T] = \
-                -(rho * area * cp * (t_m - tvor) * (1 / delta_t) + rho * area * cp * v_init * (
-                            -t_init_i + t_init_i1 - tl) / length
-                  - alpha * (t_amb - t_m) + qext)
-
-            branch_component_pit[:, JAC_DERIV_DT] = - rho * area * cp * v_init / length + alpha \
-                                                    + rho * area * cp / delta_t
-            branch_component_pit[:, JAC_DERIV_DT1] = rho * area * cp * v_init / length + 0 * alpha \
-                                                     + rho * area * cp / delta_t
-
-            branch_component_pit[:, JAC_DERIV_DT_NODE] = rho * v_init \
-                                                         * branch_component_pit[:, AREA]
-            branch_component_pit[:, LOAD_VEC_NODES_T] = rho * v_init \
-                                                        * branch_component_pit[:, AREA] * t_init_i1
+                -(rho * area * cp * (t_m - tvor) * (1 / delta_t) * length + rho * area * cp * v_init * (
+                            -t_init_i + t_init_i1 - tl)
+                  - alpha * (t_amb - t_m) * length + qext)
+
+            branch_component_pit[:, JAC_DERIV_DT] = - rho * area * cp * v_init
```

Comment:
Why can we just drop the alpha part here?

Reply:
That's the main reason for this PR, the correction of the derivatives. Since the heat losses are calculated based on the outlet temperature (I did this explicitly in another commit, but here you can see that tm=t_init_i1), it cannot be part of the partial derivative for the inlet temperature, it just gets 0 there.